File: Task340/publisher/src/main/java/com/example/publisher/dto/StoryResponseTo.java
package com.example.publisher.dto;

import java.time.Instant;
import java.util.List;

public record StoryResponseTo(
        Long id,
        Long userId,
        String title,
        String content,
        Instant created,
        Instant modified,
        List<String> labels  // ← возвращаем имена меток
) {}

File: 351001/Orlovich/Dc/Application/DTOs/Response/StoryResponseTo.cs
namespace Domain.Models;

public class StoryResponseTo
{
    public long id { get; set; }
    public long EditorId {get; set; }
    public string title { get; set; }
    public string content { get; set; }
    public DateTime created { get; set; }
    public DateTime modified { get; set; }
}